modyn/trainer_server/internal/trainer/remote_downsamplers/remote_gradnorm_downsample.py

```diff
@@ -0,0 +1,38 @@
+from typing import Any
+
+import torch
+from modyn.trainer_server.internal.trainer.remote_downsamplers.abstract_remote_downsample_strategy import (
+    AbstractRemoteDownsamplingStrategy,
+)
+
+
+class RemoteGradNormDownsampling(AbstractRemoteDownsamplingStrategy):
+    """
+    Method adapted from
```

Comment:
Is it adapted or identical? If adapted, can you add a sentence explaining what is different from the paper?

Reply:
Other optimisations are also proposed in the paper, but do not make sense in the context of dynamics datasets. This reproduces the basic version.

Reply:
ok I meant whether you changed the basic version from the paper or not. but if i understand you correctly, you didn't :)